README.md

```
@@ -2,3 +1,4 @@
+![Markdown Lab](docs/assets/github-banner.svg)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)